picard/tagger.py

```diff
@@ -208,7 +208,7 @@ def __bool__(self):
         return bool(self.commands or self.files or self.mbids or self.urls)
 
     def __str__(self):
-        return "files: %r mbids: %r urls: %r commands: %r" % (self.files, self.mbids, self.urls, self.commands)
+        return f"files: {self.files}  mbids: f{self.mbids}  urls: {self.urls}  commands: {self.commands}"
```

Comment:
Minor thing, but replacement  for `"%r" % obj` is `f"{repr(obj)}` (or `f"{obj!r}'`).
Not sure this change was intentional, in many cases the result is the same, but not always.

But anyway using Python3 formatted strings is better, so +1 from me.

Reply:
Right, thanks